### PR TITLE
Feature/content moderation social media and roles

### PR DESCRIPTION
**Pull request by robert-arias**
# Content moderation, roles, and social media share links.

## Jira issue:

- None.

## How to test

- [ ] Execute the following commands: `composer install`, `npm i`, `chirripo drush cim -- -y`, `chirripo drush cr`.
- [ ] Go to `/admin/people/create` and create a new user with the `Editor` role.
- [ ] Go to `/admin/people/create` and create a new user with the `Aprobador` role.
- [ ] Log in as a user with the `Editor` role.
- [ ] Create a `Noticia` content type.
- [ ] Save the content as `draft`, other options shouldn't appear.
- [ ] Go to `/noticias` and check if the newly created content appears in the view. It shouldn't.
- [ ] Log out.
- [ ] Log in as a user with the `Aprobador` role.
- [ ] Go to `/admin/content/moderated` and check if the content created by the `Editor` user is there.
- [ ] Edit the content created by the `Editor` user and save it as `Published`.
- [ ] Go to `/admin/content/moderated` and check if recently edited content is there. It shouldn't.
- [ ] Go to `/noticias` and check if the  content appears in the view. It should.
- [ ] Check if the `Noticia` has social media links to share the news itself.
- [ ] Share the news, choose any social media.
- [ ] Edit the published content and save it as `Unpublished`.
- [ ] Go to `/noticias` and check if the  content appears in the view. It shouldn't.

## AC

- [ ] The user with the _editor_ role should only be able to save nodes as `Draft`.
- [ ] The user with the _aprobador_ role can save content as `Draft`, `Published`, and `Unpublished`. It's in charge of changing the content status created by the user with the _editor_ role from `Draft` to `Published` (besides the administrator).
- [ ] Anyone can share the news with the social media links.
- [ ] New users can be created with the `Editor` and `Aprobador` role.

### CODE REVIEW

File: config/sync/system.action.user_add_role_action.aprobador.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+uuid: eea3bc96-e02a-486c-b523-4bb081cdfc2b
+langcode: en
+status: true
+dependencies:
+  config:
+    - user.role.aprobador
```
Comment (rigoucr) [inline config/sync/system.action.user_add_role_action.aprobador.yml:6]: All machine names should be in English

File: config/sync/user.role.aprobador.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+uuid: baea68c4-1316-475a-a259-1f87d083a4d4
+langcode: en
+status: true
+dependencies: {  }
+id: aprobador
```
Comment (rigoucr) [inline config/sync/user.role.aprobador.yml:5]: All the machine names should be in English by standard

File: config/sync/workflows.workflow.creacion_noticias.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+uuid: 43d83d99-ebd0-4d49-9044-be1542012337
+langcode: en
+status: true
+dependencies:
+  config:
+    - node.type.noticia
+  module:
+    - content_moderation
+id: creacion_noticias
```
Comment (rigoucr) [inline config/sync/workflows.workflow.creacion_noticias.yml:9]: All the machine names should be in English by standard

**Comment by rigoucr** [comment]
it works like a charm @robert-arias!! well done! for the next time just be more careful setting up the machine names for the config elements.
:shipit:

**Review by rigoucr — APPROVED** [review]

**Comment by robert-arias** [comment]
> it works like a charm @robert-arias!! well done! for the next time just be more careful setting up the machine names for the config elements.
> :shipit:

Thank you! And yes, I will be more careful, won't happen again.

**Pull request merged**